wp-content/themes/engage/.gitignore

```diff
@@ -0,0 +1 @@
+dist/*
```

Comment:
We need to leave the `dist` directory in git, as with our current set-up, we wouldn't be able to push to WP Engine. I'm looking into using codeship to build our dist directory when we deploy so we can ignore the dist, but for now, leave if in please.